suite-common/validators/src/schemas/sendFormSchema.ts

```diff
@@ -0,0 +1,121 @@
+import { G } from '@mobily/ts-belt';
+import BigNumber from 'bignumber.js';
+
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { formatNetworkAmount, isAddressValid } from '@suite-common/wallet-utils';
+import { FeeInfo } from '@suite-common/wallet-types';
+
+import { yup } from '../config';
+
+export type SendFormFormContext = {
+    networkSymbol?: NetworkSymbol;
+    availableAccountBalance?: string;
+    networkFeeInfo?: FeeInfo;
+};
+
+const isAmountDust = ({
```

Comment:
The validation is inspired by desktop (packages/suite) implementation. My original idea was to extract it to these new functions and the import it to desktop. But the desktop implementation is so messed up that the inputs and outputs of these new validation functions would have to be way more complicated than it is now. For this reason I decided to change the desktop implementation.

On the other hand if the desktop will refactor the send form one day, they can easily use this scheme from since it is defined in suite-common. That is the reason why I lest it here in suite-common close to the sendFormValidationScheme that will be potentially reused by desktop one day. What do you think?

Reply:
Keep it in native, if they will refactor they can copy it to common. Better keep relevant things together.

Reply:
moved in dab2acb8b2